### PR TITLE
Improve styles of the `BitPersona` component (#1841)

### LINKED ISSUE #1841: `BitPersona` component styling issues
In the styles of the `BitPersona` component, the unit used for some properties is `px`. Based on the styling conventions of the Bit Components, we must use the `rem` unit instead.
Another issue is that there are some colors in Hex format used in the styling of this component. We must use just the color variables declared in the `colors.fluent.scss` files.

### PR DESCRIPTION
**Pull request by saeedhalab**
This closes #1841

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/colors.fluent.scss [hunk 1]
```diff
@@ -41,4 +41,13 @@ $Orange180: #d83b01;
 $ErrorBlockRed: #FDE7E9;
 $ErrorBlockIconRed: #a80000;
 
-$Yellow20: #fff4ce;
\ No newline at end of file
+$Yellow20: #fff4ce;
+
+//Presence
+$magenta: #b4009e;
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/colors.fluent.scss:47]: use Pascal casing for the color naming.
please rename this color to `Magneta`.

File: src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/colors.fluent.scss [hunk 2]
```diff
@@ -41,4 +41,13 @@ $Orange180: #d83b01;
...
+$Gray110: #8A8886;
+$DoNotDisturbRed: #C50F1F;
+$OnlineGreen: #6bb700;
+$BusyRed: #C43148;
+$orange10: #ffaa44;
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/colors.fluent.scss:52]: please rename this color to `Orange10`.

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@maryamhdr plz submit your suggestion for the color names.

**Review by maryamhdr — APPROVED** [review]

**Comment by maryamhdr** [comment]
> @maryamhdr plz submit your suggestions for the color names.

We had discussed the names before with @saeedhalab.

**Review by msynk — APPROVED** [review]

**Pull request merged**